### DISCUSSION TITLE
Add remove function to useLocalStorage / useSessionStorage

### DISCUSSION THREAD

**Discussion by RubyHuntsman** [juliencrn/usehooks-ts#558] [Ideas]
Hey 👋 
It would be nice to have a function that removes saved values 🤔 

# Original
`const [value, setValue] = useLocalStorage('test-key', 0)`

# Feature request
`const [value, setValue, remove] = useLocalStorage('test-key', 0)`

**Comment by k-melnychuk** [comment]
I was also thinking about this...
+1

**Reply by RubyHuntsman** [reply]
I've seen this in different hook library and I think there is no reason to not add `remove` function 😄

**Reply by k-melnychuk** [reply]
I've opened a [pull request](https://github.com/juliencrn/usehooks-ts/pull/573). Hopefully, it gets reviewed and approved soon 😌

**Reply by RubyHuntsman** [reply]
Cool 😄 thanks for implementation ❤️

**Comment by juliencrn** [comment]
Merged! Thanks
